### DISCUSSION TITLE
User is blocked

### DISCUSSION THREAD

**Discussion by Henry-3000** [vercel/vercel#10517] [Help]
### Question

I tried signing up and got blocked instantly

**Comment by noohncoffee** [comment]
:white_check_mark: I sent the project files and photos I would use to registration@vercel.com and they contacted me and solved the problem. Thanks. This process may take a few days...

**Reply by ronakexe** [reply]
Same

**Comment by VicvekSr9485** [comment]
> ### Question
> I tried signing up and got blocked instantly

I'm having this same issue

**Comment by ghostfly23333** [comment]
same issue for me
